src/app/(landing)/@faq/page.tsx

```diff
@@ -1,5 +1,207 @@
+'use client';
+
+import Text from "@/components/Text";
+import Flex from "@/components/Flex";
+import Button from "@/components/Button";
+import Link from "next/link";
+
+import styles from './faq.module.scss';
+
+import { useState } from 'react';
+
+interface faqQuestions {
+  question: string | JSX.Element;
+  answer: string | JSX.Element;
+}
+
+const general = [
```

Comment:
If we can consolidate this into one object/array thingy - the UI code would be cleaner

Reply:
im lazy but sure